packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -53,7 +50,7 @@ const createTestProps = props => ({
 
 const testLocaleData = {
   isLoading: false,
-  language: 'en',
+  locale: 'en',
```

Comment:
I'm wondering what the deal is with renaming language to locale since AFAIK this is a language and not a locale?

Reply:
The renaming was for two reasons:
- to fix the inconsistencies
- to technically be correct, as `fr-FR` is a locale and not a language anymore